RSSDP/DisposableManagedObjectBase.cs

```diff
@@ -72,7 +72,7 @@ public string BuildMessage(string header, Dictionary<string, string> values)
         /// <para>Sets the <see cref="IsDisposed"/> property to true. Does not explicitly throw an exception if called multiple times, but makes no promises about behaviour of derived classes.</para>
         /// </remarks>
         /// <seealso cref="IsDisposed"/>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1063:ImplementIDisposableCorrectly", Justification="We do exactly as asked, but CA doesn't seem to like us also setting the IsDisposed property. Too bad, it's a good idea and shouldn't cause an exception or anything likely to interfer with the dispose process.")]
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1063:ImplementIDisposableCorrectly", Justification = "We do exactly as asked, but CA doesn't seem to like us also setting the IsDisposed property. Too bad, it's a good idea and shouldn't cause an exception or anything likely to interfer with the dispose process.")]
```

Comment:
Is this suppression valid? I mean, it seems to turn off a warning, maybe that warning actually makes sense to be fixed?

Reply:
It isn't, but this is in the RSSDP repo, that whole thing is a mess imo